CHANGES/7718.feature

```diff
@@ -0,0 +1,3 @@
+Fixed keep-alive connections stopping a graceful shutdown.
```

Comment:
@Dreamsorcerer “fixed” implies a “bugfix” changelog note type, but the filename says “feature”.  If it's both, it might be worth splitting into two fragments.